packages/block-editor/src/components/global-styles/typography-utils.js

```diff
@@ -31,7 +31,6 @@ import { getComputedFluidTypographyValue } from '../font-sizes/fluid-utils';
  * @property {?number} minFontSizeFactor How much to scale defaultFontSize by to derive minimumFontSize. Optional.
  * @property {?string} minFontSize       The smallest a calculated font size may be. Optional.
  */
-
```

Comment:
Nit: is this whitespace removal intended? It's good visually if there's a single whitespace line between a `typedef` comment and the JSDoc preceding a function.

Reply:
I think that was me just hacking around and forgetting to revert. Thanks! I'll change.